src/rsz/src/Resizer.cc

```diff
@@ -629,33 +629,35 @@ void Resizer::getPins(Instance* inst, PinVector& pins) const
 
 void Resizer::SwapNetNames(odb::dbITerm* iterm_to, odb::dbITerm* iterm_from)
 {
-  //
-  // The concept of this function is we are moving the name of the net
-  // in the iterm_from to the iterm_to.  We preferentially use
-  // the modnet name, if present.
-  //
-  dbNet* to_db_net = iterm_to->getNet();
-  odb::dbModNet* to_mod_net = iterm_to->getModNet();
+  if (iterm_to && iterm_from) {
```

Comment:
You need to throw an error using the Logger in the else case. I would also invert the condition and add it as a precondition instead.

If (!to || !from) { error }

Reply:
Hi, Ethan,
This is a case where we are creating a buffer and we know the pins (buffer_ip_pin_iterm/op_pin_oterm)will be valid. 
I have rewritten that part of the code to try to clean it up to use an assert.
You are right: we don't want to be propagating around loads of unnecessary pointer tests. So I simply remove the tests